README.md

```diff
@@ -29,11 +28,11 @@ The project contains both Java and C++ modules. If you are only interested in th
 the Java part only. If you're also interested in the MPI computation manager, you need to compiler C++ modules also.
 
 To build the Java modules, you need the following requirements:
-- JDK *(1.8 or greater)*
-- Maven *(3.3.9 or greater)*
+- JDK *(17 or greater)*
+- Maven *(3.8.0 or greater)*
 
 ```
-$> yum install java-1.8.0-openjdk maven 
+$> yum install java-1.17.0-openjdk maven 
```

Comment:
This is not the java17 package name:
```suggestion
$> yum install java-17-openjdk maven 
```
(see [this page for RHEL](https://access.redhat.com/documentation/en-us/red_hat_build_of_openjdk/17/html-single/installing_and_using_red_hat_build_of_openjdk_17_on_rhel/index#installing-jre-on-rhel-using-yum_openjdk))